assets/js/index.js

```diff
@@ -1,5 +1,5 @@
 // ES6 imports
-let { messages, variables } = require('./messages.json');
+let { messages, variables, categories } = require('./messages.json');
```

Comment:
`messages.json` doesn't exist, we use `messages.yml` now. But does that matter?

Reply:
nvm got confused

Reply:
When the tool is written, `messages.yml` is used, but when it's built, the yml is converted to `messages.json`. :)